lesson9/lesson9.js

```diff
@@ -0,0 +1,44 @@
+const ul = document.getElementById('js-list');
+const listItems = [
+  { to: "bookmark.html", img: "bookmark-icon.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "mail-icon.png", alt: "画像2", text: "メッセージ" }
+];
+
+function showLoadingImg() {
```

Comment:
Not sure if you need to create a function to display/remove loading image. For me, I just appended/removed the image to/from DOM, rather than calling them as a function.

Reply:
Thank you for your comment. I felt that the content was not so complete as to create a function. I removed this function and wrote the code directly.
https://github.com/rikokko/to_be_frontend/pull/17/commits/89d057a129ace8e76ff92fe4f9f992fd68f8bb89

Reply:
Removed the image display / delete function from fetchListData (). Changed to delete the image of createList (). How about that?
https://github.com/rikokko/to_be_frontend/pull/17/commits/89d057a129ace8e76ff92fe4f9f992fd68f8bb89